src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -48,7 +48,7 @@ export class GithubAppsLogsComponent implements OnInit {
   isExpansionDetailRow = (i: number, row: Object) => row.hasOwnProperty('detailRow');
 
   constructor(
-    @Inject(MAT_DIALOG_DATA) public matDialogData: string,
+    @Inject(MAT_DIALOG_DATA) public matDialogData: any,
```

Comment:
Try not to use `any`. I know we use this pattern for dialog data in several places, but our ultimate goal one day is to get rid of all `any`'s, so we should avoid adding new ones.